components/GameMenu/GameMenu.tsx

```diff
@@ -49,30 +55,90 @@ const styles = StyleSheet.create({
     }
 });
 
-const subButtons = [
+interface menuButtonDate {
```

Comment:
Need to change interface name

Reply:
Changed interface name to MenuButtonData.  Could maybe just use the type of the button props.  Change for the future.